### ISSUE TITLE
WC Priority 2 Tax taxing Priority 1 tax

### ISSUE THREAD

**Issue by sprecherbrewery** [woocommerce/woocommerce#28328]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

**Issue Description:**

- I am running WC 4.7, just recently updated from 3.1
- WP 5.6.3
- PHP 7.0.33-0+deb9u10
- Platform Debian 9.5/Apache

We are using the built-in tax module and are having an issue with Standard Rates. We have state taxes set as priority 1 taxes, and then approximate county taxes through an aggregation of zip codes to create priority 2 tax levels. When a taxable item has a non-zero rate for the level 2 tax, the tax that gets calculated is based on the sum of the sku price + the level 1 (state) tax. So, for example, an item priced at $24.40 with a level 1 (state) tax of 5% and a level 2 (zip/county) tax of .5% gets charged $1.22 in level 1 tax and $0.13 in level 2 tax. The correct amount of level 2 tax should be $0.12. The "_line_tax_data" field from a current such item looks like this:

> a:2:{s:5:"total";a:3:{i:53;s:1:"0";i:51;s:4:"1.22";i:104;s:6:"0.1281";}s:8:"subtotal";a:3:{i:53;s:1:"0";i:51;s:4:"1.22";i:104;s:6:"0.1281";}}

The "_line_tax_data" field from a current such an item on the previous version of WC looked like this:

> a:2:{s:5:"total";a:3:{i:53;s:1:"0";i:51;s:4:"1.22";i:98;s:5:"0.122";}s:8:"subtotal";a:3:{i:53;s:1:"0";i:51;s:4:"1.22";i:98;s:5:"0.122";}}

If you take the .1281 from _line_tax_data as it is now (which was .122 under the previous WC version) and divide by .005 you get 25.62 as the taxable amount instead of 24.40, and 25.62 is the sum of the the sku price and the level one tax.

So, TLDR, WC is calculating tax on a tax as far as I can tell.

Let me know what I can do to help with this.

Tim

**Comment by sprecherbrewery** [comment]
Looks like the update flipped my level 2 tax rates to compound. So its a non-issue
